replicatedhq/ship
# Add kustomize sub command

What I Did
------------
- Add a `kustomize` subcommand

How I Did it
------------
- Add a new key `HelmChartMetadata` to `ReleaseMetadata` that's resolved from the provided chart path in the kustomize subcommand
- Add a new endpoint `helm-metadata` to serve resolved `HelmChartMetadata`

How to verify it
------------
- Run subcommand with `file` flag pointed to a valid helm chart on local fs
- Curl against `%{API_ENDPOINT}/helm-metadata` should return with the following structure:
```
{
  "metadata": {
    "description": ...
    "version": ...
    "icon": ...
    "release": ...
  }
}
```

Description for the Changelog
------------
Add initial implementation of `kustomize` subcommand


Picture of a Boat (not required but encouraged)
------------
![](https://vignette.wikia.nocookie.net/onepiece/images/4/41/Going_Merry_Infobox.png/revision/latest?cb=20121214234157)
